services/dispatcher/session.hpp

```diff
@@ -36,14 +36,13 @@ class session_t {
     ///components::session::session_id_t session_;
 };
 
+// TODO Remove this session and use logic from collections
+// Move session logic from collections to a separate place
 struct session_key_t {
```

Comment:
Left structure with only one filed - not  a great idea, but otherwise I have to do some general session refactoring.
@kotbegemot